ui/actions/nuxeo-add-to-collection-button.js

```diff
@@ -244,18 +244,20 @@ import '../nuxeo-button-styles.js';
       const label = item.displayLabel || item.title;
 
       // if we are adding a new entry with the _newEntryFormatter we don't want to escape the HTML
-      return item.id === -1 ? `<iron-icon icon="nuxeo:add" item-icon></iron-icon>${label}` : escapeHTML(label);
+      return item.id === -1
+        ? `<iron-icon icon="nuxeo:add" item-icon></iron-icon>${escapeHTML(label)}`
+        : escapeHTML(label);
     }
 
     _selectionFormatter(item) {
       const label = item.displayLabel || item.title;
 
       // if we are adding a new entry with the _newEntryFormatter we don't want to escape the HTML
-      return item.id === -1 ? label : escapeHTML(label);
+      return item.id === -1 ? escapeHTML(label) : escapeHTML(label);
```

Comment:
the if and else condition is the same. Should ternary operator be required ?

Reply:
this was just for testing I had reverted this change as this is not required anymore